client/app/components/WikiPoetryApp.react.js

```diff
@@ -0,0 +1,21 @@
+//Components are like Views
+//Top level app view
+var React = require('react');
+var Navbar = require('./navbar/Navbar.react')
+var Home = require('./home/Home.react')
+var Article = require('./article/Article.react')
+
+var WikiPoetryApp = React.createClass({
+
+  render: function() {
+    return (
+      <div className="wikiapp">
+        <Navbar />
+        <Home />
+        <Article />
```

Comment:
Shouldn't we create another react class that contains the navbar and article instead of putting the article in here? Or maybe keep the Navbar in here, but then have a <Content /> tag that will diplay either the <Home /> or <Article />.